clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/DeployHandlerNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.netflix.spinnaker.clouddriver.deploy;
+
+public class DeployHandlerNotFoundException extends RuntimeException {
```

Comment:
I know its unrelated to java -> groovy but worth flipping this to a SpinnakerException while in here?